examples/sycl/pvc/pvc_bfloat_dpas_gemm_cute.cpp

```diff
@@ -206,8 +208,8 @@ struct ExampleRunner {
 
     // Check if output from CUTLASS kernel and reference kernel are relatively equal or not
     // need to set a larger error margin for comparison to succeed
-    auto epsilon = static_cast<ElementOutput>(0.1f);
-    auto nonzero_floor = static_cast<ElementOutput>(0.1f);
+    auto epsilon = static_cast<ElementOutput>(0.5f);
+    auto nonzero_floor = static_cast<ElementOutput>(0.5f);
```

Comment:
Could you check if it works using `0.1` and `2 * (rand() / double(RAND_MAX)) -1` as initialisation?